### PR TITLE
 add dimension director to step 3

### PR DESCRIPTION
**Pull request by beaugogh**

### CODE REVIEW

File: src/app/models/table-models/dimension.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export class Dimension {
```
Comment (ewelinagr) [inline src/app/models/table-models/dimension.ts]: I thought of a bit different model for dimension - to include index, label, maybe elements as well (to match Jan's response format a bit). I'm not sure what does selected mean?

File: src/app/models/table-models/dimension-zone-enum.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum DimensionZoneEnum {
+  Rows = 'Rows',
+  Columns = 'Columns',
+  Candidates = 'Candidates'
+}
```
Comment (ewelinagr) [inline src/app/models/table-models/dimension-zone-enum.ts]: What exactly is included in Candidates? Does it mean all dimensions possible and then some of them are in rows, some columns and some nowhere or these are just the ones that are not assigned to rows and columns?

File: src/app/models/table-models/dimension.ts (hunk 1 above)
Comment (beaugogh) [reply src/app/models/table-models/dimension.ts]: yes, you can add extra fields to Dimension of course

Comment (beaugogh) [reply src/app/models/table-models/dimension.ts]: 'selected' is a flag to indicate if the dimension is selected or not

File: src/app/models/table-models/dimension-zone-enum.ts (hunk 2 above)
Comment (beaugogh) [reply src/app/models/table-models/dimension-zone-enum.ts]: the candidates ware all the dimensions available from the previous step, the user will assign candidates to rows and columns to compose the table. And indeed, the user can pick a subset of candidates to assign to rows and columns

**Pull request merged**